src/components/Home.jsx

```diff
@@ -1,18 +1,119 @@
-import React, { useEffect } from 'react'
-import { fetchData } from '../redux/data/dataSlice'
-import { useDispatch } from 'react-redux'
+import React from 'react';
+
+import { useNavigate } from 'react-router-dom';
+import { useDispatch, useSelector } from 'react-redux';
+import { setSelected, setSearch } from '../redux/data/dataSlice';
 
 function Home() {
+  const dispatch = useDispatch();
+
+  const {
+    data, search, isLoading,
+  } = useSelector((data) => data.data);
+  const navigate = useNavigate();
 
- 
+  function handleChange(valor) {
+    if (valor === 'ALL') {
+      dispatch(setSearch(data));
+      return;
+    }
+    // setIsSearching(true);
```

Comment:
- [ ] kindly remove the commented code as well.

Reply:
done